### PR TITLE
Add custom url prefix support

### PR DESCRIPTION
**Pull request by rubenaprikyan**
- Created templating mechanism, to identify the path from url, and support custom url prefix
- Added Colab integration

### CODE REVIEW

**Comment by rubenaprikyan** [comment]
Resolves #1117 #213

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,9 @@
 - Add Scatter Explorer to visualize correlations between metric last value and hyperparameter (KaroMourad)
 - Add ability to track and visualize plotly objects (devfox-se, Hamik25, rubenaprikyan)
 - Add ability to query distributions by step range and density (VkoHov, rubenaprikyan)
+- Add colab notebook support (mihran113, rubenaprikyan)
 - Implement images visualization tab in run detail page (VkoHov, KaroMourad)
+- Add custom URL prefix support (mihran113, Hamik25)
```
Comment (mihran113) [inline CHANGELOG.md:12]: I guess we need to add @roubkar to this list as well :)

**Review by mihran113 — APPROVED** [review]
Everything on backend side looks good, just a comment there on changelog :)

**Review by alberttorosyan — COMMENTED** [review]
left one small comment.
the rest looks great

**Review by roubkar — APPROVED** [review]

File: aim/ext/notebook/notebook.py [hunk 2]
```diff
@@ -69,13 +69,15 @@ def get_argument_options(line):
...
         '--host': '127.0.0.1',
-        '--port': '43800'
+        '--port': '43800',
     }
     for arg in args[1:]:
         key, value = arg.split('=', 1)
         if key in supported_args:
             options[key] = value
 
+    options['--base-path'] = '/notebook'
```
[the remark this answers is not in the document]
Comment (rubenaprikyan) [reply aim/ext/notebook/notebook.py]: Fixed!

**Review by alberttorosyan — APPROVED** [review]

**Pull request merged**